sparkle/src/components/ScrollArea.tsx

```diff
@@ -48,48 +44,33 @@ const ScrollArea = React.forwardRef<
 
     const shouldHideDefaultScrollBar = hideScrollBar || hasCustomScrollBar;
 
-    const handleScroll = React.useCallback(
-      (event: React.UIEvent<HTMLDivElement>) => {
-        onScroll?.(event);
-
-        if (viewportRef.current) {
-          const scrollTop = viewportRef.current.scrollTop;
-          const isNowScrolled = scrollTop > 0;
-
-          if (isNowScrolled !== isScrolled) {
-            setIsScrolled(isNowScrolled);
-            onScrollStateChange?.(isNowScrolled);
-          }
-        }
-      },
-      [isScrolled, onScroll, onScrollStateChange]
-    );
-
     return (
       <ScrollAreaPrimitive.Root
         ref={ref}
-        className={cn("s-relative s-z-20 s-overflow-hidden", className)}
+        className={cn(scrollAreaVariants({ appearance, className }))}
         {...props}
       >
-        <ScrollAreaPrimitive.Viewport
-          ref={viewportRef}
-          onScroll={handleScroll}
-          className={cn(
-            "s-h-full s-w-full s-rounded-[inherit]",
-            isScrolled && "s-scroll-active",
-            viewportClassName
-          )}
-        >
+        <ScrollAreaPrimitive.Viewport className="s-h-full s-w-full s-rounded-[inherit]">
           {children}
         </ScrollAreaPrimitive.Viewport>
+
         {!shouldHideDefaultScrollBar && (
-          <ScrollBar orientation={orientation} className={scrollBarClassName} />
+          <ScrollAreaPrimitive.ScrollAreaScrollbar
+            orientation="vertical"
+            className={cn(
+              "s-flex s-touch-none s-select-none s-transition-opacity s-duration-200",
+              "s-opacity-0 group-hover:s-opacity-100"
+            )}
+          >
+            <ScrollAreaPrimitive.ScrollAreaThumb className="s-relative s-flex-1 s-rounded-full s-bg-muted-foreground/40 hover:s-bg-muted-foreground/70" />
```

Comment:
Need to add the dark mode classes 🙏🏻 
You have `s-bg-muted-foreground/40 hover:s-bg-muted-foreground/70`
You need to add `dark:s-bg-muted-foreground-night/40 dark:hover:s-bg-muted-foreground-night/70`

Reply:
🙈